zulipterminal/ui_tools/views.py

```diff
@@ -953,7 +953,10 @@ def __init__(self, controller: Any, msg: Message, title: str) -> None:
         msg_info = [
             ('', [('Date & Time', time.ctime(msg['timestamp'])[:-5]),
                   ('Sender', msg['sender_full_name']),
-                  ('Sender\'s Email ID', msg['sender_email'])]),
+                  ('Sender\'s Email ID', msg['sender_email']),
+                  ('Edit History', 'Press {} to view'.format(
+                      ', '.join(map(repr, keys_for_command('EDIT_HISTORY'))))),
+                  ]),
```

Comment:
Previously, this label extension bit was in the 3rd/'Introduce EditHistoryView...' commit.